packages/frontend/src/js/app/components/map/map.js

```diff
@@ -881,82 +890,75 @@ export default {
                 .classList[action]("mapPin--street");
         },
 
-        // Fonction de chargement des données geoJson régionales
+        // Fonction de chargement des marqueurs régionaux
         loadRegionalData() {
             this.regionalLayer.getLayers().forEach(layer => {
                 if (layer instanceof L.Marker) {
                     layer.remove();
                     return;
                 }
-
-                // Calcul de la position du marqueur
-                // On utilise la méthode pointOnFeature(), qui garantit que le point soit dans le polygone, plutôt que centroid()
-                const { feature } = layer;
-                const markerPosition = pointOnFeature(feature);
-                // Création du marqueur à partir de la long et lat retournées par pointOnFeature()
-                const lon = markerPosition.geometry.coordinates[0];
-                const lat = markerPosition.geometry.coordinates[1];
+            });
+            Object.keys(this.numberOfShantytownsBy.regions).forEach(key => {
```

Comment:
Tu pourrais probablement clarifier cette boucle en récupérant toutes les propriétés de `this.numberOfShantytownsBy.regions[key]` au début de chaque itération. Ça rendrait la lecture du reste de l'algo plus facile (notamment l'appel à circleWithText).

Un truc du genre :
```
const { sites: nbSites, latitude, longitude, chieftown } = this.numberOfShantytownsBy.regions[key];
```